src/FileSystemUtilities.js

```diff
@@ -65,6 +65,14 @@ export default class FileSystemUtilities {
 
   @logger.logifySync
   static unlinkSync(filePath) {
-    fs.unlink(filePath);
+    fs.unlinkSync(filePath);
+  }
+
+  @logger.logifySync
+  static isSymlink(path) {
+    const lstat = fs.lstatSync(path);
```

Comment:
I think this will throw if there's nothing at `path`.  Should this try/catch or guard with `pathExists.sync()`?